spec/controllers/chargeback_controller_spec.rb

```diff
@@ -77,21 +68,17 @@
       controller.instance_variable_set(:@layout, "chargeback")
       controller.instance_variable_set(:@_params, :id => node)
       controller.send(:tree_select)
-      response.should render_template('layouts/_saved_report_paging_bar')
-      controller.send(:flash_errors?).should_not be_true
-      expect(response.status).to eq(200)
+      expect(response).to                        render_template('layouts/_saved_report_paging_bar')
+      expect(controller.send(:flash_errors?)).to be_false
+      expect(response.status).to                 eq(200)
     end
   end
 
   render_views
 
   context "#explorer" do
-    before(:each) do
-      session[:settings] = {}
-      EvmSpecHelper.create_guid_miq_server_zone
-    end
-
     it 'can render the explorer' do
```

Comment:
You can probably drop this pointless `it` as well and just make the `context` the `it`